sale_order_product_assortment_quick/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,2 @@
+* PyTech SRL <info@pytech.it>
+* Ooops404 <info@ooops404.com>
```

Comment:
same here.
```suggestion
* renda-dev <YOUR@EMAIL>
```

Reply:
hi @mmequignon , I think it's optional to choose whether to publish name and/or email of contributor?

Reply:
@pedrobaeza could you please confirm?
Thanks!

Reply:
Emails have always been optional (we don't put them, as we don't want spam from leechers asking for free support). I don't have a strong opinion here putting the company or the specific contributor. We put specific contributors as a recognition to them, and in the open source spirit. But it's also something that can go against you. If a weak developer put something in public not in the same quality level than the rest of the company releases, whole company is blamed without distinctions.

Reply:
Of course nobody has prevented the contributor to add their name in the readme file, but if they prefer to remain anonymous I don't see where's the problem